projects/storefrontlib/src/cms-components/myaccount/order/amend-order/cancellations/cancel-order/cancel-order.component.spec.ts

```diff
@@ -6,6 +6,7 @@ import { OrderEntry } from '@spartacus/core';
 import { of } from 'rxjs';
 import { OrderAmendService } from '../../amend-order.service';
 import { CancelOrderComponent } from './cancel-order.component';
+import { FormErrorsModule } from '../../../../../../../../../dist/storefrontlib/shared/components/form/form-errors/index';
```

Comment:
Your import uses your built app. Please change it to the following:
```suggestion
import { FormErrorsModule } from '../../../../../../shared/index';
```